lib/ansiblelint/rules/__init__.py

```diff
@@ -245,11 +245,37 @@ def __repr__(self) -> str:
                           for rule in sorted(self.rules, key=lambda x: x.id)])
 
     def listtags(self) -> str:
+        tag_desc = {
+            "behaviour": "Indicates a bad practice or behavior",
+            "bug": "Likely wrong usage pattern",
+            "command-shell": "Specific to use of command and shell modules",
+            "core": "Related to internal implementation of the linter",
+            "deprecations": "Indicate use of features that are removed from Ansible",
+            "experimental": "Newly introduced rules, by default triggering only warnings",
+            "formatting": "Related to code-style",
+            "idempotency":
+                "Possible indication that consequent runs would produce different results",
+            "idiom": "Anti-pattern detected, likely to cause undesired behavior",
```

Comment:
I'm not sure what an anti-pattern would be... but if your users know, then ignore my comment :-)